code/game/turfs/simulated/walls_misc.dm

```diff
@@ -50,6 +50,11 @@
 	icon = 'icons/turf/walls/coated_reinforced_wall.dmi'
 	max_temperature = INFINITY
 
+/turf/simulated/wall/r_wall/swarm_proof  // Used to stop swarmers messing with atmos mixes
+	name = "reinforced wall"
```

Comment:
All the values below are dupes from the actual r_wall and are not required due to how inheritance works

Reply:
So just leave it blank?

Reply:
Yep yep. Just a blank `/turf/simulated/wall/r_wall/swarm_proof` will do. Can even do without but it's cleaner to at least have it defined somewhere and not just in the proc